doc/user_manual/rom.tex

```diff
@@ -639,6 +639,18 @@ \subsubsection{SciKitLearn}
 \end{itemize}
 Based on the \xmlNode{SKLtype} several different algorithms are available.
 %
+In addition, the temporal vairable for a \textbf{HistorySet} \xmlString{SciKitLearn}
+is specified in \xmlNode{pivotParameter}:
+\begin{itemize}
+  \item \xmlNode{pivotParameter}, \xmlDesc{string, optional parameter}, specifies the pivot
```

Comment:
is this explaination available for any time dep rom? It is the same for all and not just for SciKitLearn

Reply:
added.